frontend/src/components/Home.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import ResponsiveAppBar from "./ResponsiveAppBar";
+
+function Home() {
+   
+        return (
+        <div>
+          <nav>
+            <ResponsiveAppBar />
+          </nav>
+          <div>
+            <h1>Home</h1>
+          </div>
+        </div>
+        );
+    }
+  
+
+
+export default Home;
+
```

Comment:
Nit but it looks like there's a buncha extra lines here, I'll look into adding a code formatter so this stuff gets cleaned up